Makefile

```diff
@@ -5,6 +5,7 @@ HERE = $(shell pwd)
 PTYPE=pypy
 ifneq ($(PTYPE), python)
     REQS=$(PTYPE)-requirements.txt
+    export with_gmp=no
```

Comment:
was this the reason? https://github.com/dhermes/oauth2client/commit/9b07670103447f5678f13f10bf5fe7387fc0d8ea